site/src/components/indicator/forceIndex.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: component
+title: Force Index
+component: indicator/algorithm/forceIndex.js
+tags:
+namespace: indicator
+
+example-code: |
+
```

Comment:
Can you remove this blank line? I've just realised it's appearing in the output :(